kubernetes.go

```diff
@@ -72,6 +79,7 @@ type NodePool struct {
 	AutoScaler   bool              `json:"auto_scaler"`
 	Tag          string            `json:"tag"`
 	Labels       map[string]string `json:"labels"`
+	Taints       []Taint           `json:"taints,omitempty"`
```

Comment:
I think `omitempty` here is only going to lead to invalid references by a consumer?  It's better to just let it be `nil`

Reply:
Ahh, I thought I needed it for when it's null